Fundamentals/Setup/Upgrading/umbraco10-breaking-changes.md

```diff
@@ -0,0 +1,149 @@
+
+# Breaking changes
+
+Below it is possible to read about the breaking changes between Umbraco 9 and 10.
+The article is subject to change, the closer we get to the launch of Umbraco 10.
+
+## Update 'diff' from 3.5.0 to 5.0.0
+
+diff library used in the Backoffice client is outdated by two major versions. Updating this will be a breaking change since the exposed global object has been renamed from JsDiff to Diff.
+
+## V10/feature/content schedule performance
+
+Removes mutable ContentSchedule property from IContent/Content to read/write content schedules please make use of _IContentService.GetContentScheduleByContentId && IContentService.PersistContentSchedule_ or the optional _contentSchedule parameter_ on _IContentService.Save_
+
+## v10 misc - Removed some redundant event handling code
+
+- Removed public methods: _PublishedSnapshotServiceEventHandler.Dispose_, _PublishedSnapshotServiceEventHandler.Dispose(bool)_ and _.PublishedSnapshotServiceEventHandler.Initialize_
+- Removed public ctor.
+
+## v10 scope provider cleanup
+
+- Some public classes in the _Cms.Core.Services_ namespace have moved assembly from **_Umbraco.Cms.Infrastructure_** to **_Umbraco.Cms.Core_**
+
+- These same public classes have changed namespace from **_Umbraco.Cms.Core.Services.Implement_** to **_Umbraco.Cms.Core.Services_**.
+
+## v10 update to npoco5
+
+NPoco types and interfaces are part of our public interface and there are changes upstream.
+
+## v10 SQLite support + distributed locking abstractions
+
+- Removed support for Microsoft SQL Server Compact (SQL CE)
+
+- Removed ReadLock & WriteLock methods from _ISqlSyntaxProvider_ interface, please instead make use of - _IDistributedLockingMechanism_ (or IScope which delegates to _IDistributedLockingMechanism_).
+
+- Constants for SQL Server provider name moved+consolidated from _Core.Constants.DatabaseProviders_ & _Core.Constants.-DbProviderNames_ to _Umbraco.Cms.Persistence.SqlServer.Constants_
+
+- Various SQL Server related services moved from the Umbraco.Infrastructure project to the new Umbraco.Cms.
+Persistence.- SqlServer project with altered namespaces e.g.
+
+- _SqlServerSyntaxProvider_, _SqlServerBulkSqlInsertProvider_, _SqlServerDatabaseCreator_
+
+**Added various methods / properties to ISqlSyntaxProvider, these must be implemented in any downstream implementation e.g:**
+
+- _ISqlSyntaxProvider.HandleCreateTable(IDatabase,TableDefinition,Boolean)_
+- _ISqlSyntaxProvider.GetFieldNameForUpdate()_
+- _ISqlSyntaxProvider.GetColumn(DatabaseType,String,String,String,String,Boolean)_
+- _ISqlSyntaxProvider.InsertForUpdateHint(Sql)_
+- _ISqlSyntaxProvider.AppendForUpdateHint(Sql)_
+- _ISqlSyntaxProvider.LeftJoinWithNestedJoin(Sql,Func<Sql,Sql>,String)_
+
+## v10: Update to ImageSharp v2
+
+**Update dependency versions**:
+
+- _SixLabors.ImageSharp from 1.0.4 to 2.1.1 (see breaking changes in v2.0.0 release notes);_
+
+- _SixLabors.ImageSharp.Web from 1.0.5 to 2.0.0 (see breaking changed in release notes);_
```

Comment:
```suggestion
- _SixLabors.ImageSharp.Web from 1.0.5 to 2.0.0 (see breaking changes in release notes);_
```

Reply:
We should probably just remove the "See breaking changes" part as it is not linking to it anyways